src/app/components/ImageTable.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useEffect } from 'react';
+import { SearchInput, Title, Toolbar, ToolbarContent, ToolbarItem, Pagination } from '@patternfly/react-core';
+import { Table, Thead, Tr, Th, ThProps, Tbody, Td } from '@patternfly/react-table';
+import { fetch } from 'cross-fetch'
+
+interface ImageData {
+  name: string;
+  version: string;
+  imageId: string;
+  provider: string;
+  region: string;
+  arch: string;
+  date: string;
+  selflink: URL;
+  virt: string;
+};
+
+export const ImageTable: React.FunctionComponent = () => {
+  const [search, setSearch] = React.useState('');
+  const [imageData, setImageData] = React.useState([]);
+  const [activeSortIndex, setActiveSortIndex] = React.useState<number | undefined>(undefined);
+  const [activeSortDirection, setActiveSortDirection] = React.useState<'asc' | 'desc' | undefined>(undefined);
+  const [page, setPage] = React.useState(1);
+  const [perPage, setPerPage] = React.useState(20);
+  const columnNames = {
+    name: 'Name',
+    provider: 'Provider',
+    region: 'Region',
+    arch: 'Architecture',
+    date: 'Date'
```

Comment:
Should be make this `Creation date` or something a bit more specific?

Reply:
Yeah makes sense. Would that be creation date or release date?

Reply:
Release date works for me.